src/utils.ts

```diff
@@ -136,11 +136,25 @@ export const recursivelyDownloadTx = async (
 
   // Transaction was already confirmed by a different block
   if (meta.first_block && meta.first_block !== blockId) {
-    const firstBlockResponse = await downloadTx(meta.first_block);
+    let firstBlockHeight = meta.first_block_height;
+
+    // We need a fallback for old fullnodes as the first_block_height attribute was only introduced
```

Comment:
Do we really need a fallback?

I mean, we are sure that the wallet-service's fullnode is on v0.49.0 already.

Unless someone else from the community is running the wallet-service-daemon out there, maybe we could even remove this fallback logic.